Open_Science_New

```diff
@@ -123,39 +107,40 @@
 	    <table class="reveal">
 	      <tr>
 		<td width="47%" style="vertical-align: middle">
-		  <img src="images/Nature_open_science.png"> 
+		<a href="https://www.nature.com/articles/533S71a"><img src="images/Nature_open_science.png"></a>
 		</td>
 		<td width="53%" style="vertical-align: middle">
-		  <img src="images/Open_science.png"> 
+		<a href="https://www.sciencemag.org/news/2016/01/montreal-institute-going-open-accelerate-science"><img src="images/Open_science.png"></a>   
 	    </td></tr></table>	
 	  </section>
 
 	  <!-- CONP -->
-	  <section data-background="images/CONP.png" data-background-size="100%">
+	  <section>
+         <a href="https://conp.ca"><img src="images/CONP.png" data-background-size="100%"></a>
 	  </section>
 
 	  <!-- TOSI -->
 	  <section>	
-	    <img src="images/TOSI.png">
+        <a href="https://www.mcgill.ca/newsroom/channels/news/mcgill-university-announces-transformative-20-million-donation-montreal-neurological-institute-and-264838"><img src="images/TOSI.png"></a>
 	  </section> 
 
 	  <!-- MNI Open Science CyberInfrastructure -->
 	  <section>		
-	    <h2><Demo</h2>
-		   <img src="images/Open_science_paper.png">
+	    <h2>Demo</h2>
+		   <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5216036/"><img src="images/Open_science_paper.png"></a>
```

Comment:
This is the link to the actual paper that NIH page points to : 
https://www.frontiersin.org/articles/10.3389/fninf.2016.00053/full

Reply:
thanks, done!